netbox_docker_plugin/forms/log_driver_option.py

```diff
@@ -0,0 +1,28 @@
+"""Log Driver option Form definition"""
+
+from django import forms
+from utilities.forms.fields import DynamicModelChoiceField
+from ..models.container import LogDriverOption, Container
+
+
+class LogDriverOptionForm(forms.ModelForm):
```

Comment:
You should extend `utilities.forms.mixins.BootstrapMixin` for having the good form style.

```suggestion
class LogDriverOptionForm(BootstrapMixin, forms.ModelForm):
```